ui/service.go

```diff
@@ -325,32 +336,33 @@ func (gs *uiService) ListGroups(ctx context.Context, token string) ([]byte, erro
 }
 
 func (gs *uiService) ViewGroup(ctx context.Context, token, id string) ([]byte, error) {
-	tpl, err := template.ParseGlob(templateDir + "/*")
-	if err != nil {
-		return []byte{}, err
-	}
-	group, err := gs.sdk.Group(id, "123")
+	tpl := template.New("groups")
+	tpl = tpl.Funcs(template.FuncMap{
+		"toJSON": func(data map[string]interface{}) string {
+			ret, _ := json.Marshal(data)
+			return string(ret)
+		},
+	})
+	var err error
```

Comment:
I do not like moving this here, I prefer it on the top of the fnction. Also, I do not like declaring `err` separately, I think this should be declared at the same time as `tpl`, as it was before.

Is there any special need to introduce `tpl := template.New("groups")`? Would notit be picked up via parsing globally in template dir?